### ISSUE TITLE
In Java IconSpinnerItem can be created just with 9 parameters

### ISSUE THREAD

**Issue by waver11** [skydoves/PowerSpinner#48]
In version 1.1.6 IconSpinnerItem can be created just with 9 parameters. 

```
ArrayList<IconSpinnerItem> iconSpinnerItems = new ArrayList<>();
        iconSpinnerItems.add(new IconSpinnerItem("Spain", ContextCompat.getDrawable(getActivity(), R.drawable.spain),null,null,0,null,null,null,null));
        iconSpinnerItems.add(new IconSpinnerItem("US", ContextCompat.getDrawable(getActivity(), R.drawable.unitedstates),null,null,0,null,null,null,null));
        iconSpinnerItems.add(new IconSpinnerItem("UK", ContextCompat.getDrawable(getActivity(), R.drawable.spain),null,null,0,null,null,null,null));
```
In this implementation Icons are not visible in dropdown!
What should be passed as arguments to constructor so icon will be visible in dropdown? 
Thanks for help

**Comment by skydoves** [comment]
Hi, I just build the demo project with Java code but it works fine for me.
And you don't need to pass all parameters to the `IconSpinnerItem `.

```java
 new IconSpinnerItem("Canada", ContextCompat.getDrawable(context, R.drawable.canada));
```

Furthermore, you should check the adapter is the `IconSpinnerAdapter`.

**Comment by waver11** [comment]
please check latest version com.github.skydoves:powerspinner:1.1.6

I found out that everything works as expected in version 1.1.5

**Comment by skydoves** [comment]
Hmm, I think there was some shipping error when publishing.
I will check it soon!

**Comment by skydoves** [comment]
@waver11 
Just published a new version `1.1.7`!
Thanks for your issue :)
